protos/nanorpc.proto

```
@@ -1,48 +1,213 @@
+// Delivery Guarantees:
+// - Requests: Guaranteed response (success or error)
+// - Updates: Best-effort delivery, no acknowledgement required
+// - Ordering: Updates maintain send order per subscription
+// - Concurrency: Thread-safe publishing allows concurrent updates
+//
+// See NANORPC_PROTOCOL.md for complete protocol specification.
+
 syntax = "proto3";
 
+import "google/protobuf/descriptor.proto";
 import "nanopb.proto";
```

Buf (1.55.1)

import "nanopb.proto": file does not exist

(COMPILE)